### PR TITLE
Adding botcity-maestro-sdk.

### PR DESCRIPTION
**Pull request by hhslepicka**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hhslepicka** [comment]
@conda-forge/help-python this one is ready for a review/merge. Thank you.

File: recipes/botcity-maestro-sdk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - requests
```
Comment (mwcraig) [inline recipes/botcity-maestro-sdk/meta.yaml:21]: Can you please drop `requests` from the `host` requirements? I'm pretty sure it isn't needed at build time.

**Review by mwcraig — COMMENTED** [review]
Thanks for opening this PR. Just one minor is needed.

File: recipes/botcity-maestro-sdk/meta.yaml (hunk 1 above)
Comment (hhslepicka) [reply recipes/botcity-maestro-sdk/meta.yaml]: @mwcraig thanks for the review. I addressed your comment via https://github.com/conda-forge/staged-recipes/pull/15265/commits/347d9ecd357c386d74802475f7237b0cc6d313a0

**Comment by mwcraig** [comment]
Thanks again, @hhslepicka, merging!

**Pull request merged**